aspnet/security/authentication/cookieauthentication.rst

```diff
@@ -0,0 +1,149 @@
+.. _security-authentication-cookie-middleware:
+
+Using Cookie Middleware without ASP.NET Identity
+================================================
+
+ASP.NET v5 provides cookie :ref:`middleware <fundamentals-middleware>` which serializes a user principal into an encrypted cookie and then, on subsequent requests, validates the cookie, recreates the principal and assigns it to the ``User`` property on ``HttpContext``. If you want to provide your on login screens and user databases you can use the cookie middleware as a standalone feature.
```

Comment:
Ah okay, I thought this was a general introduction, not a paragraph specific to your sample :smile: